### PR TITLE
Vega addition to telluric (and test_telluric)

### PR DESCRIPTION
**Pull request by michaelcushing**
Please god work.

### CODE REVIEW

File: tests/test_telluric.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import glob
+import os
+import pyspextool as ps
+
+
+ps.pyspextool_setup(raw_path='./',
```
Comment (kelle) [inline tests/test_telluric.py:6]: Usually the tests should be in a function.

File: src/pyspextool/telluric/telluric_correction.py [hunk 2]
```diff
@@ -141,6 +158,16 @@ def telluric_correction(object_file, standard, standard_file, output_name,
...
+    if verbose is True:
+        logging.getLogger().setLevel(logging.INFO)
+        setup.state["verbose"] = True
+        
+    elif verbose is False:
+        logging.getLogger().setLevel(logging.ERROR)
+        setup.state["verbose"] = False
+
+    logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+    logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
```
Comment (kelle) [inline src/pyspextool/telluric/telluric_correction.py:170]: You shouldn't need these logging config statements. We should set this up somewhere else which is called by everything else.  At least put one of these lines outside of the function.

File: src/pyspextool/telluric/vega.py [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+from pyspextool.io.check import check_parameter
+from pyspextool.utils.arrays import find_index
+from pyspextool.fit.fit_peak1d import fit_peak1d
+from pyspextool.telluric import config as telluric
+from pyspextool.plot.limits import get_spec_range
+
+def vega_xcorrelate(object_wavelength, object_flux, vega_wavelength,
+                    vega_flux, vega_continuum, minimum_wavelength=None,
+                    maximum_wavelength=None, resolving_power=None,
+                    qa_show=False, qa_show_plotsize=(6,10), qa_fileinfo=None):
```
Comment (kelle) [inline src/pyspextool/telluric/vega.py:15]: consider doing the typehinting in the function definition.

File: tests/test_telluric.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+png_files = glob.glob(os.path.join("tests/test_data/misc_data/","test.png"))
+
+assert len(fits_files) == 1
+assert len(png_files) == 1
+
+# CLEANUP
+    # remove generated files
+for files in fits_files + png_files:
+    os.remove(files)
+
```
Comment (kelle) [inline tests/test_telluric.py:30]: Consider putting a place holder test for vega.py

File: tests/test_telluric.py (hunk 1 above)
Comment (michaelcushing) [reply tests/test_telluric.py:6]: Done.

File: src/pyspextool/telluric/telluric_correction.py (hunk 2 above)
Comment (michaelcushing) [reply src/pyspextool/telluric/telluric_correction.py:170]: Done.

File: src/pyspextool/telluric/vega.py (hunk 3 above)
Comment (michaelcushing) [reply src/pyspextool/telluric/vega.py:15]: Done.

**Comment by kelle** [comment]
You gotta push the changes....

**Comment by michaelcushing** [comment]
Yup.  Still working on the test for Vega.py.

File: tests/test_telluric.py (hunk 4 above)
Comment (michaelcushing) [reply tests/test_telluric.py:30]: Added (it works, but it is just written to work)

**Review by kelle — APPROVED** [review]
Looks great!

**Pull request merged**